target/linux/armsr/armv7/config-6.12

```diff
@@ -14,12 +14,14 @@ CONFIG_ARCH_SELECT_MEMORY_MODEL=y
 CONFIG_ARCH_VIRT=y
 CONFIG_ARM=y
 CONFIG_ARM_CPU_SUSPEND=y
+# CONFIG_ARM_DEBUG_WX is not set
```

Comment:
This is not needed any more, it was added in aaf89c2c2b2ca8575b367816bc5081eca403a1ca